profiler/profile.go

```diff
@@ -32,6 +35,10 @@ const (
 	MutexProfile
 	// GoroutineProfile reports stack traces of all current goroutines
 	GoroutineProfile
+	// GoroutineWaitProfile reports stack traces and wait durations for
+	// goroutines that have been waiting or blocked by a syscall for > 1 minute
+	// since the last GC.
+	GoroutineWaitProfile
```

Comment:
@pmbauer we should discuss how we'll roll this out internally. Should we perhaps not expose any public APIs for now and make this feature only available via an env variable or sth?

Reply:
That is a good point.
We can hide this with an env var, make this const un-exported, and hide the UI support for this profile type behind a feature flag

Reply:
Done. 4bccddff0b854a800e7e33b540853c61477f537c - PTAL